### PR TITLE
tuna: init at 0.15

### PR DESCRIPTION
**Pull request by elohmeier**
###### Motivation for this change
Tuna is a useful tool to analyze and modify Linux threads.

###### Things done
Added Tuna and dependent Python libraries.

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/python-ethtool/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  version = "0.14";
+
+  src = fetchFromGitHub {
+    owner = "fedora-python";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Sp7ssfLZ/1FEKrvX257pKcaureZ5mdpJ7jCEh/ft1l0=";
+  };
+
+  patchPhase = ''
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-ethtool/default.nix:24]:
```suggestion
  postPatch = ''
```

File: pkgs/development/python-modules/python-ethtool/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    substituteInPlace tests/parse_ifconfig.py --replace "Popen('ifconfig'," "Popen('${nettools}/bin/ifconfig',"
+  '';
+
+  buildInputs = [ libnl ];
+  nativeBuildInputs = [ pkg-config ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-ethtool/default.nix:29]: Missing pythonImportsCheck.

File: pkgs/development/python-modules/python-ethtool/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  meta = with lib; {
+    description = "Python bindings for the ethtool kernel interface";
+    homepage = "https://github.com/fedora-python/python-ethtool";
+    license = licenses.gpl2Plus;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-ethtool/default.nix:34]: Missing maintainer.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/python-ethtool/default.nix [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools_scm
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-ethtool/default.nix:4]:
```suggestion
```

File: pkgs/development/python-modules/python-ethtool/default.nix [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+, fusepy
+, fuse
+, openssl
+, pkg-config
+, libnl
+, nettools
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-ethtool/default.nix:10]:
```suggestion
, pkg-config
, nettools
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/python-ethtool/default.nix (hunk 5 above)
Comment (elohmeier) [reply pkgs/development/python-modules/python-ethtool/default.nix]: Thanks, updated.

**Pull request merged**